django/db/models/expressions.py

```diff
@@ -620,6 +624,8 @@ def as_sql(self, compiler, connection):
 
 
 class RawSQL(Expression):
+    contributes_to_group_by = True
```

Comment:
Okay, I guess there should be a test for that last commit ("Respect contributes_to_group_by in src expressions" too?